gix-traverse/tests/traverse.rs

```diff
@@ -5,4 +5,5 @@ fn hex_to_id(hex: &str) -> gix_hash::ObjectId {
 }
 
 mod commit;
+mod topo;
```

Comment:
Tests must follow the package structure, so it should be in `commit/topo`.

Reply:
Hmm... Can you clarify how would want this solved? [Currently](https://github.com/Byron/gitoxide/blob/main/gix-traverse/tests/commit/mod.rs) there is only a `commit/mod.rs` which contains `mod ancestors { ... }` inline. Should I split that out into `commit/ancestors.rs` and then add `commit/topo.rs` or just append `mod topo { ... }`?

Reply:
Sorry for the confusion, don't worry about it - I will do it as part of my refactoring which might change things around even more. A lot is intuition by now and I just have to find a moment to focus on this PR to get it merged.